typescript/packages/subsurface-viewer/src/storybook/examples/MultiViewExamples.stories.tsx

```diff
@@ -72,15 +72,15 @@ export const MultiViewAnnotation: StoryObj<typeof SubsurfaceViewer> = {
                 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                 /* @ts-expect-error */
                 <View id="view_1">
-                    <ContinuousLegend min={-3071} max={41048} />
+                    <ContinuousLegend min={2724} max={3396} />
                     <ViewFooter>kH netmap</ViewFooter>
                 </View>
             }
             {
                 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                 /* @ts-expect-error */
                 <View id="view_2">
-                    <ContinuousLegend min={2725} max={3396} />
+                    <ContinuousLegend min={-1863} max={40798} />
```

Comment:
It looks like the left view is showing the Hugin depth values as a property and the right is showing the kH netmap property. Can you fix the view annotations as well?

Also, I think you set the wrong min value on the property range for the right-hand view?

Reply:
Nope :)
The kH netmap minimum is not the same. And the ViewFooter annotations are the same as before (the left property color looks like depth values, but is annotated "kH netmap".